### ISSUE TITLE
Loss suddenly dropping to almost zero

### ISSUE THREAD

**Issue by kritchie** [tensorflow/models#3783]
Hi guys,

I need some insights for a problem I'm having. I'm trying to retrain an ssd_mobilenet_v1 architecture with a custom dataset. I'm having issues about training loss dropping suddenly to something close to 0.

```
...
INFO:tensorflow:global step 444: loss = 13.0682 (0.573 sec/step)
INFO:tensorflow:global step 445: loss = 11.3450 (0.564 sec/step)
INFO:tensorflow:global step 446: loss = 11.8106 (0.569 sec/step)
INFO:tensorflow:global step 447: loss = 42.5687 (0.577 sec/step)
INFO:tensorflow:global step 448: loss = 0.5660 (0.574 sec/step)
INFO:tensorflow:global step 449: loss = 0.5660 (0.562 sec/step)
INFO:tensorflow:global step 450: loss = 0.5660 (0.556 sec/step)
INFO:tensorflow:global step 451: loss = 0.5660 (0.572 sec/step)
INFO:tensorflow:global step 452: loss = 0.5660 (0.570 sec/step)
INFO:tensorflow:global step 453: loss = 0.5660 (0.572 sec/step)
INFO:tensorflow:global step 454: loss = 0.5660 (0.569 sec/step)
INFO:tensorflow:global step 455: loss = 0.5660 (0.569 sec/step)
...
```

I tried extracting the images and bounding boxes from the TFRecord file and visually inspecting them and it looks fine. I'm not seeing anything wrong with my annotations. So I'm wondering why the loss is suddenly dropping to zero. 

Did any of encountered any issue like that ? 

Edit : I've noted that it always happen at the same step. (E.g. Step 288 always around 0.5 ). So this would tell me that something is wrong with the records within my tfrecord file.

**Comment by kritchie** [comment]
After reviewing, the labelling was wrong on those particular entries. That's why the loss was 0. 
Data set labelling was the issue.
